### ISSUE TITLE
[Bug]: Components page accessibility issues

### ISSUE THREAD

**Issue by pya35** [Orange-OpenSource/ods-ios#563]
### Prerequisites

- [X] I have [searched the backlog](https://github.com/Orange-OpenSource/ods-ios/issues) for duplicate or closed feature requests

### Your test device

iPhone 14 Plus

### OS Version

17.1.1

### App version

v0.16.0

### Describe the issue

This list is about "Components" page

- [x] #652

- [x] Truncated component names in zoom 235%
![image](https://github.com/Orange-OpenSource/ods-ios/assets/105045667/4f269757-ba4b-4080-b49b-6af45c8322e7)
**Recommandation: Check that Dynamic Type is correctly applied**

- [x] Keyboard focus contrast <3:1.
![image](https://github.com/Orange-OpenSource/ods-ios/assets/105045667/54946bb2-ff62-4e4d-82af-63fa1c10988c)
**Recommandation: If possible, add an outline to make the focus more visible or reinforce the focus with darker orange.**

- [x] No vocalization of title after displaying the page
**Recommandation: Use announcements (notifications) to force the title page vocalization - To be done on each main page Guidelines, Components, Modules & About**

- [x] Landscape mode from components root page leads to open "Bar-tabs" page
**Recommandation: no content change on landscape mode**

### Expected Behavior

Components page = 100% accessible (updated status 20/12/2023: 70%)

**Comment by pya35** [comment]
Landscape mode from components root page leads to open "Bar-tabs" page : seems to be fixed on last release

**Comment by pylapp** [comment]
@pya35 What do you mean by "Low text and icon contrast: orange on grey"? Could you provide some screenshots please?

**Comment by pya35** [comment]
@pylapp all orange texts on light gray or white background have a contrast 3:1 which is too low for mobile use. Expected is 4.5:1.
Advice : use black or dark orange text. To be checked with design team if possible.
![image](https://github.com/Orange-OpenSource/ods-ios/assets/105045667/064fe328-98c7-4343-a526-4c82451f4abd)
![image](https://github.com/Orange-OpenSource/ods-ios/assets/105045667/95094571-b17a-4e62-813a-36ebd1143074)

**Comment by pylapp** [comment]
@pya35 
- About "Low text and icon contrast: orange on grey", it is a design issue, DSM must be updated
- About "Keyboard focus contrast <3:1.", filed to get some API and inputs from documentation to get the focus state updated on elements if keyboard is uses, won't fix yet

cc @B3nz01d  @mccart77

**Comment by pya35** [comment]
keyboard focus visibility can be solved with high contrast in Full Keyboard Access settings

**Comment by pya35** [comment]
OK
